src/Reservations/SFA.DAS.Reservations.Api/AppStart/AddServiceRegistrationExtension.cs

```diff
@@ -19,6 +19,7 @@ public static void AddServiceRegistration(this IServiceCollection services)
             services.AddTransient<ICoursesApiClient<CoursesApiConfiguration>, CourseApiClient>();
             services.AddTransient<ICourseDeliveryApiClient<CourseDeliveryApiConfiguration>, CourseDeliveryApiClient>();
             services.AddTransient<IAccountsApiClient<AccountsConfiguration>, AccountsApiClient>();
+            services.AddTransient<IEmployerFinanceApiClient<EmployerFinanceApiConfiguration>, EmployerFinanceApiClient>();
```

Comment:
IFinanceApiClient - as it matches IAccountsApiClient without the Employer prefix

Reply:
resolved the changes in https://github.com/SkillsFundingAgency/das-apim-endpoints/pull/834